sbt

```diff
@@ -229,8 +231,13 @@ execRunner () {
     vlog ""
   }
 
-  [[ -n "$batch" ]] && exec </dev/null
-  exec "$@"
+  setTrapExit
+
+  if [[ -n "$batch" ]]; then
+    "$@" < /dev/null
+  else
+    "$@"
```

Comment:
I once dug into the history of the use of exec in sbt-extras.

It's been there a very long time: https://github.com/paulp/sbt-extras/commit/1c84c7fca18cc6bcecf6857ef0574d83de5a2000 (Jan 2012), the there was a brief period where it was removed: https://github.com/paulp/sbt-extras/commit/f8a021375d6d6cfc38c9ce05816785344d9373a8 (Mar 2013), and then put back: https://github.com/paulp/sbt-extras/commit/229a2e6aa9e7dd8535720acf7fa1c4803fc43063 (Sep 2013).

Is the removal here required?